cmd/buildkitd/config/config.go

```diff
@@ -21,7 +21,7 @@ type Config struct {
 		Containerd ContainerdConfig `toml:"containerd"`
 	} `toml:"worker"`
 
-	Registries map[string]resolver.RegistryConfig `toml:"registry"`
+	Registries map[string]rconfig.RegistryConfig `toml:"registry"`
```

Comment:
nit: can we call this `resolverconfig`. I think the extra verbosity is justified here

Reply:
done